test/profiler/test_profiler.py

```diff
@@ -2357,31 +2358,31 @@ def EventTreeDFS(event_tree):
             with open(json_file_path, "w") as f:
                 json.dump([kineto_events, profiler_events], f)
 
-        assert (os.path.exists(json_file_path))
-        with open(json_file_path, "r") as f:
-            kineto_events, profiler_events = json.load(f)
+            assert (os.path.exists(json_file_path))
```

Comment:
Why move this inside the condition? This only runs in accept mode now!

Reply:
well when accept is False, the test fails:
```
Traceback (most recent call last):
  File "pytorch/test/profiler/test_profiler.py", line 2374, in test_utils_get_optimizable_events
    basic_evaluation = _utils.BasicEvaluation(self.load_mock_profile())
  File "pytorch/test/profiler/test_profiler.py", line 2240, in load_mock_profile
    kineto_events, profiler_events = json.load(f)
  File "/usr/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/usr/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.8/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 13 (char 12)
```
the new test works:
<img width="550" alt="Screen Shot 2023-02-01 at 8 20 02 AM" src="https://user-images.githubusercontent.com/20074092/216100855-92be0482-633c-494e-b805-fc721c92ed53.png">

Reply:
But the test succeeds on master today right?

Reply:
@puririshi98 Looking at this a bit closer, did this fail for you because accept was False, or because the test went through the accept block and produced garbage that json couldn't decode? Was this test run with a build using `USE_KINETO=0`? It's possible that we don't see this fail on master because master builds with `USE_KINETO=1`. If Jetson can't be built with Kineto, may be that's a whole different issue, and we can utilize one of your skip decorators.

Reply:
however, the jetson wheels are being built w/ USE_KINETO=0 for now. so maybe this should just be skipped then?

Reply:
```
riship@ubuntu:~/pytorch$ python3 test/profiler/test_profiler.py -v -k test_utils_get_optimizable_events
test_utils_get_optimizable_events (__main__.TestExperimentalUtils) ... accept = False
[W kineto_shim.cpp:320] Adding profiling metadata requires using torch.profiler with Kineto support (USE_KINETO=1)
[W kineto_shim.cpp:320] Adding profiling metadata requires using torch.profiler with Kineto support (USE_KINETO=1)
```

Reply:
should just be skipped instead